src/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+}
+
+.main-heading,
+.heading,
+.paragraph {
```

Comment:
actually, for this task there was no need to use classes, you could just use tag-names for selectors, but ok)